README.md

```diff
@@ -15,78 +17,221 @@
+[Installation](#installation)&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;[Documentation](http://intel.github.io/daal/)&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;[Examples](#examples)&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;[Get Help](https://software.intel.com/en-us/forums/intel-data-analytics-acceleration-library)&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;[How to Contribute](CONTRIBUTING.md)&nbsp;&nbsp;&nbsp;
 
-[![Build Status](https://dev.azure.com/daal/DAAL/_apis/build/status/oneapi-src.oneDAL?branchName=master)](https://dev.azure.com/daal/DAAL/_build/latest?definitionId=5&branchName=master) ![License](https://img.shields.io/github/license/oneapi-src/oneDAL.svg)
+[![Build Status](https://dev.azure.com/daal/DAAL/_apis/build/status/intel.daal?branchName=master)](https://dev.azure.com/daal/DAAL/_build/latest?definitionId=3&branchName=master) ![License](https://img.shields.io/github/license/intel/daal.svg)
 
+Intel&reg; oneAPI Data Analytics Library (oneDAL) is a powerful machine liearning library that helps speed up big data analysis.
 
-Intel(R) oneAPI Data Analytics Library (oneDAL) is a library that helps speed up big data analysis. 
-We provide highly optimized algorithmic building blocks for all stages of data analytics: **preprocessing**, **transformation**, **analysis**, **modeling**, **validation**, and **decision making**. Our algorithms suppost **batch**, **online**, and **distributed** processing modes of computation. 
-
-The current version of oneDAL provides Data Parallel C++ (DPC++) API extensions to the traditional C++ interface.
+Intel&reg; oneAPI Data Analytics Library is an extenstion of Intel&reg; Data Analytics Acceleration Library (Intel&reg; DAAL).
 
-## Table of Contents <!-- omit in toc -->
+Intel&reg; oneDAL solvers are also used in [Intel Distribution for Python]([https://software.intel.com/en-us/distribution-for-python](https://software.intel.com/en-us/distribution-for-python)) in scikit-learn optimization.
 
-- [Technical Preview Features](#preview-features)
-- [oneDAL and Intel(R) DAAL](#onedal-and-intelr-daal)
+## Table of Contents
+- [Build yours high-performance data science application with intel&reg; oneDAL](#build-yours-high-performance-data-science-application-with-intel-onedal)
+- [Technical Preview Features](#technical-preview-features)
+- [Python API](#python-api)
+- [Scikit-learn patching](#scikit-learn-patching)
+- [Distributed multi-node mode](#distributed-multi-node-mode)
 - [Installation](#installation)
+- [Installation from Source](#installation-from-source)
 - [Examples](#examples)
+- [Samples](#samples)
+- [oneDAL Apache Spark MLlib samples](#onedal-apache-spark-mllib-samples)
```

Comment:
Move acceleration of MLLib after "Distributed multi-node mode" and just add reference to "Samples" part